frontend/src/scenes/ingestion/panels/VerificationPanel.tsx

```diff
@@ -1,147 +1,66 @@
-                        <Button type="link" onClick={showInviteModal}>
-                            <UserAddOutlined />
-                            Invite team member
-                        </Button>
-                    </Menu.Item>
-                    <Menu.Item key="2" data-attr="ingestion-help-item-slack">
-                        <a href="https://posthog.com/slack?s=app" target="_blank">
-                            <Button type="link">
-                                <SlackSquareOutlined />
-                                Ask us in Slack
-                            </Button>
-                        </a>
-                    </Menu.Item>
-                </Menu>
-            )
-            return (
-                <Dropdown
-                    overlay={menu}
-                    trigger={['click']}
-                    visible={isHelpMenuShowing}
-                    onVisibleChange={(v) => {
-                        setHelpMenuShowing(v)
-                    }}
-                >
-                    <Button type="primary" data-attr="ingestion-help-button" onClick={() => setHelpMenuShowing(true)}>
-                        Need help? <DownOutlined />
-                    </Button>
-                </Dropdown>
-            )
-        }
-
-        const popoverTitle = (
-            <Space direction="vertical">
-                <Text strong>Are you sure you want to continue without data?</Text>
-                <Text>You won't be able to conduct analysis or use most tools without event data.</Text>
-                <Text>For the best experience, we recommend adding event data before continuing.</Text>
-            </Space>
-        )
-        return (
-            <Space style={{ float: 'right' }}>
-                <Popconfirm
-                    title={popoverTitle}
-                    okText="Yes, I know what I'm doing."
-                    okType="danger"
-                    cancelText="No, go back."
-                    onVisibleChange={(v) => {
-                        setPopConfirmShowing(v)
-                    }}
-                    onCancel={() => {
-                        setPopConfirmShowing(false)
-                    }}
-                    visible={isPopConfirmShowing}
-                    onConfirm={completeOnboarding}
-                    cancelButtonProps={{ type: 'primary' }}
-                >
-                    <Button
-                        type="dashed"
-                        data-attr="ingestion-continue-anyway"
-                        onClick={() => {
-                            setPopConfirmShowing(true)
-                        }}
-                    >
-                        Continue without verifying
-                    </Button>
-                </Popconfirm>
-                <HelpButton />
-            </Space>
-        )
-    }
-
     return (
         <CardContainer index={index} onBack={() => setVerify(false)}>
-            {!currentTeam?.ingested_event ? (
```

Comment:
@rcmarron thinking of just removing all of this and not even going to feature flag it because i don't think it'll impact conversion rate. What do you think?